### DISCUSSION TITLE
How to interpret the results from LinkPeaks() analysis?

### DISCUSSION THREAD

**Discussion by AndiL04** [stuart-lab/signac#1590] [Q&A] [answered]
Hi, 

I have recently been utilizing Signac for analysis on a 10x Multiome dataset. During this process, I successfully executed the LinkPeaks function on genes of interest.

However, I found difficulties to interprete some part of the results. 
1. As illustrated in the accompanying screenshot, there appears to be a discrepancy between the significantly correlated genomic regions and the identified peaks. My initial assumption was that these genomic regions would represent the Transcription Start Sites (TSS) of the targeted genes.
2. Additionally, I am unclear why the linked genomic regions vary in width.

<img width="757" alt="github" src="https://github.com/stuart-lab/signac/assets/62026346/90c41f9b-04c7-416c-80bf-5b4d7549d54f">

Thank you for your help!

Andi

**Comment by timoast** [comment, marked as answer]
The genomic ranges themselves (seqnames, start, end) give the coordinates of the link to be drawn when creating a plot, which is the midpoint of the peak and the TSS of the gene. The more informative part for an analysis is in the metadata columns: score, gene, peak, zscore, pvalue. These give the name of the gene and coordinates of the peak and the strength of the relationship. Please see the documentation for [LinkPeaks](https://stuartlab.org/signac/reference/linkpeaks) for more

**Reply by AndiL04** [reply]
Thank you so much for the explanation!
